src/components/app-login/app-login.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../assets/fontawesome/css/all.css';
+
+.login-container {
+  width: 20rem;
+  margin: auto;
+  margin-top: 20%;
+
+  .field-wrap {
+    input {
+      width: 100%;
```

Comment:
same styles are repeated below, use SCSS's & operator to avoid repetition
.field-wrap {
&-notValid{
//not valid styles
}
}